std/range/package.d

```diff
@@ -197,6 +197,14 @@ import std.traits; // CommonType, isCallable, isFloatingPoint, isIntegral,
 Iterates a bidirectional range backwards. The original range can be
 accessed by using the $(D source) property. Applying retro twice to
 the same range yields the original range.
+
+Params:
+    r = a bidirectional range
```

Comment:
hm not really happy with the description. maybe something like

r = the bidirectional range to iterate backwards